File: php/image_moderation_batch_aksk_demo.php
<?php
/**
 * 图片内容检测批量服务ak,sk 方式请求的示例
 */
require "./ais_sdk/image_moderation_batch.php";
require "./ais_sdk/utils.php";

// region目前支持华北-北京一(cn-north-1)、亚太-香港(ap-southeast-1)
init_region($region = 'cn-north-1');

$app_key = "*************";
$app_secret = "*************";

$data_url1 = "https://ais-sample-data.obs.cn-north-1.myhuaweicloud.com/terrorism.jpg";
$data_url2 = "https://ais-sample-data.obs.cn-north-1.myhuaweicloud.com/antiporn.jpg";

$result = image_content_batch_aksk($app_key, $app_secret, array($data_url1, $data_url2), array("politics", "terrorism", "porn"), 0);
echo $result;

File: python3/image_moderation_batch_aksk_demo.py
# -*- coding:utf-8 -*-
from ais_sdk.image_moderation_batch import moderation_image_batch_aksk
from ais_sdk.utils import init_global_env

if __name__ == '__main__':
    # Region currently supports Beijing(cn-north-1) and Hong Kong(ap-southeast-1)
    init_global_env('cn-north-1')

    #
    # access moderation image of batch jobs,post data by ak,sk
    #
    app_key = '*************'
    app_secret = '************'

    demo_data_url1 = 'https://ais-sample-data.obs.cn-north-1.myhuaweicloud.com/terrorism.jpg'
    demo_data_url2 = 'https://ais-sample-data.obs.cn-north-1.myhuaweicloud.com/antiporn.jpg'

    # call interface use the url
    result = moderation_image_batch_aksk(app_key, app_secret, [demo_data_url1, demo_data_url2],
                                         ['politics', 'terrorism', 'porn'], 0)
    print(result)
